cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -282,6 +294,12 @@ function Invoke-Installer($bin_paths, $platforms) {
       }
     }
   }
+  foreach ($lib_path in $artifacts["lib_paths"]) {
+    $installed_file = Split-Path -Path "$lib_path" -Leaf
+    Copy-Item "$lib_path" -Destination "$dest_dir"
+    Remove-Item "$lib_path" -Recurse -Force
+    Write-Information "  $installed_file"
```

Comment:
Probably don't want to bother informing users about installed dylibs... probably?

Reply:
Probably not, yeah. We might but we also might not?

My thought is more or less:

* If the lib is an actual downstream-usable artifact, like a C library for non-Rust software, I might want to know it's there. (Say the user packaged something like glib - it could be nice to know I got `libglib-2.0.0.dylib`.)
* If the lib is just used for a binary, or a secondary artifact like Python/Ruby/etc bindings, I probably don't want to know it's there.

Reply:
We discussed this and agreed that right now it's probably best for us to install these to begin with, and we can adjust that behaviour in future versions.